tide_site.install

```diff
@@ -740,3 +741,54 @@ function tide_site_update_8026() {
     $config->set('resourceFields', $content);
   }
 }
+
+/**
+ * New field taxonomy image logo.
+ */
+function tide_site_update_8027() {
+  $configs = [
+    'paragraphs.paragraphs_type.call_to_action_image_logo' => 'paragraphs_type',
+    'field.storage.paragraph.field_feature_image' => 'field_storage_config',
+    'field.field.paragraph.call_to_action_image_logo.field_feature_image' => 'field_config',
+    'field.field.paragraph.call_to_action_image_logo.field_paragraph_cta' => 'field_config',
+    'core.entity_form_display.paragraph.call_to_action_image_logo.default' => 'entity_form_display',
+    'core.entity_view_display.paragraph.call_to_action_image_logo.default' => 'entity_view_display',
+
+  ];
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [\Drupal::service('extension.list.module')->getPath('tide_site') . '/config/install'];
+  // Check if field already exported to config/sync.
+  foreach ($configs as $config => $type) {
+    $config_read = _tide_read_config($config, $config_location, TRUE);
+    $storage = \Drupal::entityTypeManager()->getStorage($type);
+    $id = substr($config, strrpos($config, '.') + 1);
+    if ($storage->load($id) == NULL) {
+      $config_entity = $storage->createFromStorageRecord($config_read);
+      $config_entity->save();
+    }
+  }
+
+  $update_configs = [
+    'core.entity_view_display.taxonomy_term.sites.default' => 'entity_view_display',
+    'core.entity_form_display.taxonomy_term.sites.default' => 'entity_form_display',
+    'field.field.taxonomy_term.sites.field_site_footer_logos' => 'field_config',
+    'field.storage.taxonomy_term.field_site_footer_logos' => 'field_storage_config',
+  ];
```

Comment:
Your two config files in optional are not referenced anywhere in the update hook here - are they needed to be installed via an update or original install, or should they be removed?

Reply:
Thanks @krakerag . I just updated my pr for your review.

Reply:
Thanks @yeniatencio - from what I can tell though, they are in the optional folder, so your code won't find them still because the $config_location is defined as the install folder.

You'll need to handle this in a seperate import block of code I think.

Reply:
Hey @krakerag , I just realised that those 2 files already existed and not need to be added. I removed them and all working good.